tests/Platforms/SQLServerPlatformTestCase.php

```diff
@@ -50,29 +50,6 @@ public function getGenerateTableWithMultiColumnUniqueIndexSql(): array
         ];
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    public function getGenerateAlterTableSql(): array
-    {
-        return [
-            'ALTER TABLE mytable ADD quota INT',
-            'ALTER TABLE mytable DROP COLUMN foo',
-            'ALTER TABLE mytable ALTER COLUMN baz NVARCHAR(255) NOT NULL',
-            "ALTER TABLE mytable ADD CONSTRAINT DF_6B2BD609_78240498 DEFAULT 'def' FOR baz",
-            'ALTER TABLE mytable ALTER COLUMN bloo BIT NOT NULL',
-            'ALTER TABLE mytable ADD CONSTRAINT DF_6B2BD609_CECED971 DEFAULT 0 FOR bloo',
-            "sp_rename 'mytable', 'userlist'",
-            "DECLARE @sql NVARCHAR(MAX) = N''; " .
-            "SELECT @sql += N'EXEC sp_rename N''' + dc.name + ''', N''' " .
-            "+ REPLACE(dc.name, '6B2BD609', 'E2B58069') + ''', ''json'';' " .
-            'FROM sys.default_constraints dc ' .
-            'JOIN sys.tables tbl ON dc.parent_object_id = tbl.object_id ' .
-            "WHERE tbl.name = 'userlist';" .
-            'EXEC sp_executesql @sql',
```

Comment:
Assertions made against SQL copy-pasted from the debugger don't add any value. The only thing they prove is that the generated SQL will not accidentally change which complicates changing code (including fixing bugs and refactoring.